server/routerlicious/packages/services-shared/src/http.ts

```diff
@@ -103,7 +104,7 @@ export function handleResponse<T>(
 			if (error instanceof Error && error?.name === "NetworkError") {
 				const networkError = error as NetworkError;
 				response
-					.status(errorStatus ?? networkError.code ?? 400)
+					.status(networkError.code ?? errorStatus ?? 400)
```

Comment:
How does this impact only get delta api and not other apis?

Reply:
The errorStatus override is not used anywhere in the codebase

Reply:
got it. just a nit: Maybe update the description of errorStatus parameter above on line 68's comment (can do later since already merged)

Reply:
I have the same question and not sure how the answer addresses it... isn't `handleResponse` is used by pretty much all route handlers for all components?

Reply:
That is correct..but this fix comes into play only when errorStatus is passed in..that is happening only in the getDelta calls

Reply:
Ah, got it, thanks!